src/controls/qml/Application.qml

```diff
@@ -47,13 +48,15 @@ Application_p {
         edge: Qt.RightEdge
         visible: false
         z: 10
+        anchors.verticalCenterOffset: edge === Qt.LeftEdge || edge === Qt.RightEdge ? DeviceInfo.flatTireHeight/2 : 0
     }
 
     Indicator {
         id: leftIndicator
         edge: Qt.LeftEdge
         visible: true
         z: 10
+        anchors.verticalCenterOffset: edge === Qt.LeftEdge || edge === Qt.RightEdge ? DeviceInfo.flatTireHeight/2 : 0
```

Comment:
Then, if you don't want to move this code into Indicator.qml there is no point keeping the condition. You only set the verticalCenterOffset property on the right and left indicators, you don't have to check if they are left or right and can directly set the offset to flatTireHeight/2

Reply:
oh, I didn't even realise I did that. I think I just copied over the code from where I had it in indicator.qml. Thank you for pointing this out, I'll clean it up.